readme.md

```diff
@@ -1,7 +1,8 @@
 Portfolio-Setup-and-Mobile-first Contact page.
-Project Description The Primary purpose of this project is to complete the mobile version of my portfolio website.
+Project Description The Primary purpose of this project is to complete the desktop version of my portfolio website.
 
 Objectives 
+build desktop version of portfolio website
 Build a Toolbar for the project 
 Build a headline section for the project 
 Build the About Section 
```

Comment:
- [ ] Great work documenting your project I can see that you used some of the headers from the[ README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by microverse however you can add as many headers from the template as possible including the `Getting started` header to your readme, and use markdown to style the readme a little bit.